enh(doc): add release note for MAP 22.04.1 (#1665)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.04.1

Release date: `July 13, 2022`

#### Bug fixes

> - Updated notification number field to align with central database schema

Reviewer: ```suggestion
- Updated the notification number field to match the central database schema.
```

#### Enhancements

> - diagnostic.sh script is now compatible with debian.

Reviewer: ```suggestion
- The diagnostic.sh script is now compatible with Debian.
```

> - Updated connected server label to "Configured server" for clarity purpose

Reviewer: ```suggestion
- Updated the connected server label to 'Configured server' for clarity purpose.
```

> - BBDOv3 is now enabled by default at install and after upgrade of the map server, to align with the central broker new 22.04 configuration

Reviewer: ```suggestion
- BBDOv3 is now enabled by default at installation and after the MAP server upgrade, to match the central Broker's new 22.04 configuration.
```
